core/dbt/task/printer.py

```diff
@@ -67,14 +71,11 @@ def get_counts(flat_nodes) -> str:
 
 
 def print_start_line(description: str, index: int, total: int) -> None:
-    msg = "START {}".format(description)
-    print_fancy_output_line(msg, 'RUN', logger.info, index, total)
```

Comment:
`print_fancy_output_line` is used in multiple places and needs to be replaced.

Reply:
if it's used all over the place, leaving it where it is and just importing it might be the way to go for now. I'd just put a TODO on the import statement.

Reply:
I need to tweak it slightly as it uses the `logger_fn` parameter which is dynamically determining log levels.  I moved it since it needs to be modified anyways.

Reply:
oohhhhhh good point. I didn't catch that.